LoggerProj/Logger/FileLogger.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.IO;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string filePath;
+
+        public FileLogger(string filePath)
+        {
+            this.filePath = filePath;
+
+            if (!File.Exists(filePath))
+            {
+                File.Create(filePath).Dispose();
+            }
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            string log = $"{DateTime.Now.ToString()} {ClassName} {logLevel} {message}";
```

Comment:
ToString call is not needed when inside of an interpolated string.
```suggestion
            string log = $"{DateTime.Now} {ClassName} {logLevel} {message}";
```